### PR TITLE
v2.0

### PR DESCRIPTION
**Pull request by znicholls**
# Pull request

My proposed final tweaks before releasing v2.0

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable) (N/A)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <http://link-to-pr.com>`_) Added feature which does something
- (`#XX <http://link-to-pr.com>`_) Fixed bug identified in (`#XX <http://link-to-issue.com>`_)
```

### CODE REVIEW

**Comment by znicholls** [comment]
@rgieseke I've pinned cftime temporarily to get the CI passing. I think it will work again once the windows wheel for cftime 1.3.1 is released, it just hasn't happened yet.

**Comment by rgieseke** [comment]
Examples in the Readme need to be converted ...

**Comment by znicholls** [comment]
> Examples in the Readme need to be converted

Do we want to keep them? If so, we should probably work out how to test them...

**Comment by rgieseke** [comment]
I'm fine with removing them (or some of them).

Tools could be

https://byexamples.github.io/byexample/

https://github.com/nschloe/exdown

**Comment by znicholls** [comment]
> https://github.com/nschloe/exdown

beautiful! Will have a play today

**Comment by znicholls** [comment]
> it just hasn't happened yet

Issue here, not sure how quickly it will happen... https://github.com/Unidata/cftime/issues/224

**Comment by znicholls** [comment]
> Issue here, not sure how quickly it will happen... [Unidata/cftime#224](https://github.com/Unidata/cftime/issues/224)

It appears to be a Python 3.6 issue, do we drop support?

**Comment by rgieseke** [comment]
> > Issue here, not sure how quickly it will happen... [Unidata/cftime#224](https://github.com/Unidata/cftime/issues/224)
> 
> It appears to be a Python 3.6 issue, do we drop support?

"Following the release of 3.6.8, we plan to provide security fixes for Python 3.6 as needed through 2021, five years following its initial release."
https://www.python.org/downloads/release/python-360/

Not sure how widespread 3.6 is on Linux distros as default or macOS (or in the OpenSCM community ...).
Droppping the tests on Windows for 3.6 is probably fine.

**Comment by lewisjared** [comment]
We can pin `cftime` to 1.3.0 for the time being. I wouldn't drop support Python 3.6 yet as we don't explicitly need any 3.7+ features, but happy to drop Windows Python3.6 tests

**Comment by rgieseke** [comment]
Not sure how these Windows wheels are produced, but it looks like the split is from 1.3.0 to 1.3.1

https://www.lfd.uci.edu/~gohlke/pythonlibs/#cftime

Maybe easiest to just drop the test on Windows ...

**Comment by znicholls** [comment]
> Maybe easiest to just drop the test on Windows

Fixed it I think

File: .github/workflows/ci-cd-workflow.yml [hunk 1]
```diff
@@ -86,6 +86,13 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
+    # no windows wheel for Python 3.6 cftime 1.3.1
```
Comment (rgieseke) [inline .github/workflows/ci-cd-workflow.yml:89]: Genius :-)

**Review by rgieseke — APPROVED** [review]

**Review by lewisjared — APPROVED** [review]
Brilliant! Good pickup on the rcps

**Pull request merged**